I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -411,13 +414,13 @@
    for a WDM TE-link between WDM-TE-nodes (OMS MCG). In this scenario,
    the node containing the remote optical transponders can be
    considered as WDM-TE-node with termination capability only and no
-   no switching capabilities.
+   switching capabilities.
   </t>
   
   <t>
-   An OMS MCG is terminated on both ends by a link termination point
+   An WDM TE-link is terminated on both ends by a link termination point
    (LTP) as defined in <xref target="RFC8345"/>.
-   Links in optical transport networks are typically bidirectional but
+   Links in optical transport networks can have different impairments and 
```

Comment:
I think that with the new text we miss the bidirectional issue

I would suggest to say: "Links in optical transport networks are typically bidirectional but can have different impairments in the two directions, Therefore they have to be modeled as ..."

Reply:
```suggestion
   Links in optical transport networks are typically bidirectional but can have different impairments in the two directions. Therefore they 
```